docs/The-Free-Plan.md

```diff
@@ -0,0 +1,73 @@
+---
+title: The Free Plan
+description: Everything you need to know about Expensify's Free Plan!
+---
+
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# What is the Free Plan?
+The free plan is ideal for start-ups and small businesses to manage expenses. With the Free Plan, a workspace admin can set up their team with Expensify Cards, reimburse cash expenses, send invoices, and manage bills, all for free! You will have total visibility and control over all spending associated with your workspace in real time.
+
+# Features Included with the Free Plan
+- Expensify Cards for all employees
+- Invoicing
+- Bill Pay
+- Unlimited receipt scanning for everyone in the company
+- Free next-day ACH reimbursements for cash expenses
+- Up to 1% cash back for card spending over $25,000 per month
+- Free corporate travel booking
+
+# Setting Up the Free Plan
+- Navigate to new.expensify.com, enter your company email address, and set a password
+- Click the **green “+”** button and select **_New workspace_**
+
+<img class="image" width="400" src="/assets/images/new-workspace.png">
+
+Once you’ve created your Workspace, you should receive a message from Concierge encouraging you to chat with your Setup Specialist.
+
+<img class="image" width="400" src="/assets/images/chat-with-concierge-setup.png">
```

Comment:
For these desktop screenshots I think we can change the `width` value to 800:

```suggestion
<img class="image" width="800" src="/assets/images/chat-with-concierge-setup.png">
```

This is how they look with `width=400`:

<img width="1154" alt="Screen Shot 2022-10-21 at 17 36 15" src="https://user-images.githubusercontent.com/6829422/197298754-f0d03a28-507c-4d6b-9d59-7f8789b3ffe6.png">

And this with `width=800`:

<img width="1107" alt="Screen Shot 2022-10-21 at 17 37 53" src="https://user-images.githubusercontent.com/6829422/197298831-2854cc8c-b9c1-490d-9dbb-ce5995a88ff1.png">